### ISSUE TITLE
Which of these possible "X has name" rules are WCAG requirements?

### ISSUE THREAD

**Issue by annethyme** [auto-wcag/auto-wcag#369]
According to WAI-ARIA 1.1 a bunch of items are required to have an accessible name. Not all of them are necessarily WCAG requirements too, though. 
How many of the following rules should be considered WCAG requirements? 

- Alertdialog has name
- Application has name
- Columnheader has name
- Dialog has name
- Grid has name
- Gridcell has name
- Heading has name - YES, if the heading shows up on the list of headings in a screen reader?
- Log has name
- Marquee has name
- Option has name
- Progressbar has name
- Radiogroup has name
- Region has name - YES?
- Rowheader has name
- Table has name
- Tabpanel has name
- Timer has name
- Tooltip has name
- Tree has name
- Treegrid has name
- Treeitem has name

I guess the rest could be considered "recommendation" rules.

**Comment by annethyme** [comment]
WCAG definition of user interface components: https://www.w3.org/TR/WCAG21/#dfn-user-interface-components

**Comment by annethyme** [comment]
Notes from Auto-WCAG call: 

- Focusable elements are in general user interface components and fall under 1.1.1.
- The term "controls" is undefined in WCAG, but we interpret it as something the user has control over and/or interact with, only navigate. None of these fall under this.
- E.g. a progress bar could non-text content and fall under 1.1.1 Non-Text Content.

**Comment by annethyme** [comment]
@WilcoFiers, for the sake of getting ACT Rules Format through CR, would it be interesting to write one or more WAI-ARIA requirement rules that are not WCAG requirements?
